### ISSUE TITLE
Build page for Regional Coordinator bios

### ISSUE THREAD

**Issue by hannahkane** [mozilla/learning.mozilla.org#1815]
RCs are doing awesome work promoting and supporting Clubs around the world. We need to lend legitimacy to their work by publishing a list of current RCs on the site.

My thought is that this can be a single page display a list of RCs. For each we might have:
 * Name
 * Where they are in the world
 * Photo
 * Brief (2-3 sentence) bio
 * List of Clubs they manage

@amirad, thoughts on this ^ ?

Ideally, we'd build this with the CMS, so it can be maintained by staff.

**Comment by amirad** [comment]
Huge +1 to being built with the cms since this list will change and will be much easier if it's easily manageable by staff. 

That looks good to me! In terms of navigation for all clubs pages should we consider having the main page, then having a 'where in the world are clubs page' with the RCs and the map, and then the resource repository? so three pages total?

**Comment by hannahkane** [comment]
rolling into next heartbeat, blocked on data collection

**Comment by hannahkane** [comment]
I have created this using Calypso, but it's blocked on releasing Calypso.

**Comment by hannahkane** [comment]
Reviving this ticket, but we now need to design and build it without Calypso.

Bios are [here](https://docs.google.com/document/d/16NWLHU5CAYEUucUOkb689QHYyEI--ulg5j8V7MQm4_8/edit)
Photos are [here](https://drive.google.com/a/mozilla.com/file/d/0B9NqWPTRsfNXYUNwdlJfMGVBQjZwb2RuVjBMZGRjTjZlU1lR/view?usp=sharing)

**Comment by amirad** [comment]
Looping @jvallera into this

**Comment by jvallera** [comment]
Hi @hannahkane. anything you need from me for this?

**Comment by hannahkane** [comment]
Thanks for the ping, @jvallera. I'm going to push this to the next heartbeat, because I don't think it's going to realistically happen this heartbeat.

Could you take a look at the bios linked in the comment above, and just confirm whether there have been any changes? Thanks!

**Comment by jvallera** [comment]
I would like to make a few edits to that. Would like to remove Andrea and Ani from the list and replace with Tina and Moses. @amirad sound ok to you?

**Comment by amirad** [comment]
Yes to removing both and adding Tina. If you've been talking to Moses and can get info from him (I couldn't earlier) then feel free to add him too.

**Comment by jvallera** [comment]
Great. I emailed Tina and Moses. Once I hear back I hope to replace Ani with Tina. @amirad you think it is ok to replace Andre with Moses if I hear back? or keep them both? Just want to stay current now that Andre is moving on.

**Comment by amirad** [comment]
I would keep Andre - from my discussion with him he will still be acting in the role in a volunteer way.

**Comment by jvallera** [comment]
Added Tina to the doc. Have not heard back from Moses so leaving him out for now. List is up to date now. Can move forward with creating page @hannahkane

**Comment by jvallera** [comment]
A club captain recently asked me if we had a list of RC's and the regions they are associated with. Could this be that list? In that case I would be sure that everyone on it is active and current RC. What do you think @hannahkane and @amirad

**Comment by hannahkane** [comment]
I think this should be that page. Are we not sure about whether everyone is active?

**Comment by amirad** [comment]
definitely should be this page

**Comment by jvallera** [comment]
Ok, in that case we will have to edit the document a bit. We need to take Rodrigo off (he has no clubs that I am aware of) and add info for Asma, Vishal, Dorine, Simeon, Moses and Ankit. That would be the full list of RC's that I know are currently active. Sound ok?

If so I will send requests for their information.

**Comment by hannahkane** [comment]
Sounds good to me! My expectation was that this would be a list of currently-active RCs.

**Comment by jvallera** [comment]
perfect, will add info as I get it. If anyone does not respond or declines I will leave them out of the list. Should have this info by the milestone.

**Comment by jvallera** [comment]
@amirad can you confirm Rodrigo's as active or inactive? I don't see any of his clubs listed in central station or in the database, but might be missing something.

**Comment by amirad** [comment]
I guess inactive - he wants to run clubs in school but says he can't until we have a localization strategy and portugese materials.

**Comment by jvallera** [comment]
@hannahkane what is the best way to add photos to the zip file that is linked above? I am able to download it, but not add content to it.

**Comment by hannahkane** [comment]
@jvallera - you can create a new folder (could be google drive, or Dropbox, or whatever) and link to it here. Or, I don't know, @gvn - what's the best way for @jvallera to get photos to you?

**Comment by gvn** [comment]
Just attaching a zip to this ticket would be great!

**Comment by jvallera** [comment]
it is only one pic, so here is the link https://drive.google.com/a/mozilla.com/file/d/0B5ga06pZVp55VW02N1pDSG9qZkNPZjRxczBReVBwY1B3N3M4/view?usp=sharing

**Comment by taisdesouzalessa** [comment]
Hey @jvallera, as we discussed in our meeting today, here is the info that has been missing on the page:
1. Pictures: Gauthamraj Elango, Andre Alves Garzia and Tina Verbo.
2. Names of the clubs that Andre Alves Garzia manages/Names of the clubs that Su Adams manages
3. It would be nice to have an intro text to give a bit of more context on the Regional Coordinators' role and how they support the Club Captains. It would be placed on the top of the page. Let me know what you think about that. Here is a screenshot with the placeholder copy:
<img width="438" alt="screen shot 2016-09-23 at 1 24 23 pm" src="https://cloud.githubusercontent.com/assets/22157921/18800369/6e463282-8191-11e6-85a6-b2d55c63131b.png">
4. In the RC Bios google doc, there are 2 different descriptions for Asma Islam Swapna - pages 4 and 6 of the document. Which one should I use?

Thank you!

**Comment by jvallera** [comment]
Thanks @taisdesouzalessa 
1. Pictures attached.  
2. UPdated names in google doc
3. Yes, agreed. How about "The Regional Coordinator (RC) is a person with strong leadership and mentorship skills. They work closely with the staff members and Club Captains to make sure the Clubs program is successfully running in their local region. [link here: https://docs.google.com/document/d/1GypBGnffjxfyX174g7fiLAcZF8tSKC5Xo7Qd4gpC5yM/pub]
4. I deleted one of Asma's descriptions so you should know which one to use. 

![93440b4ac8ee399cce7c532a1ea63117](https://cloud.githubusercontent.com/assets/3729312/18842933/c5c3ffda-83e4-11e6-9fd9-c7b43e9e771c.jpg)
![final-gauth](https://cloud.githubusercontent.com/assets/3729312/18842937/c8f15694-83e4-11e6-95e1-f862609e61cf.jpg)
![ctkyb5lxeaaq2-x](https://cloud.githubusercontent.com/assets/3729312/18847221/57e818bc-83f7-11e6-9f84-59fe39e71aa6.jpg)

**Comment by taisdesouzalessa** [comment]
Hi @jvallera, thanks for the assets! 

Team, here is the link for feedback on RedPen:
https://redpen.io/p/nj4f7768ca3e7b79cf

I also prepared a quick prototype in InVision so you can see the transition from one page to the other: https://invis.io/FQ8PRPS48#/192224333_R2-1300px_-_Large_Device_Desktops_-_Start_A_Club_-_V2      - click on "Veteran Regional Coordinators" text link then on the tab to go back to the initial page.
![tabbed-behaviour](https://cloud.githubusercontent.com/assets/22157921/18896901/617bc6ec-84da-11e6-92e8-47fc3cbe00f4.gif)

Please note the mobile version has just 2 entries because it is just a sample on how the data will be showed there. The mobile will have the same full list as the desktop version. Thanks!

**Comment by taisdesouzalessa** [comment]
Some notes:
1. I used Julia's copy suggestion (about who are the Regional Coordinators) in the first page - "Start a Club". So we can give some context in advance to the users. However, it would be nice to have an additional information on the bios page. For now, I added some I thought it could fit (based on the document Julia shared). Please let me know if you think there is a more suitable copy:
<img width="896" alt="screen shot 2016-09-27 at 5 53 34 pm" src="https://cloud.githubusercontent.com/assets/22157921/18897004/3dd08b46-84db-11e6-9bf0-6635bf086ce7.png">

2. I changed Asma Islam Swapna's copy to third person since all the other copy is in third person to make it consistent.

3. You'll see 2 options on RedPen: one with a group picture (we have a placeholder in black and white there now) and one with a mosaic. If we have a group picture of the RCs please send to me and I'll replace it. Let me know which option you prefer :).
<img width="772" alt="screen shot 2016-09-27 at 5 56 08 pm" src="https://cloud.githubusercontent.com/assets/22157921/18897079/d78eea48-84db-11e6-9ee2-837d862337a7.png">
<img width="777" alt="screen shot 2016-09-27 at 5 56 14 pm" src="https://cloud.githubusercontent.com/assets/22157921/18897080/d793590c-84db-11e6-9eb3-ef9576bb1f94.png">

4. Not sure if this is a common use case but if a Regional Coordinator wants to send somebody the link for this page he or she is, it would be nice to have a direct link to their name (since we don't have a search). @gvn would it be possible to add an id or name so we can have a link like "www.url.com/#name"?

5. Could we scramble the list a bit? Currently it is very separated by gender: the men are on the top of the list and the women are right after. I would like to mix the list to diversify a bit. Does the order of the names matter?

Sorry for the long comments here. I thought it was important to highlight those issues...let me know if there is a better way of adding those comments :)

**Comment by jvallera** [comment]
Thanks @taisdesouzalessa here is what I can add:

1. Change title to  "Featured Regional Coordinators" instead of "Veteran Regional Coordinators". And use this "Regional Coordinators mentor multiple Club Captains in one geographic location to realize their full potential through clear, continued leadership and support. Learn more about some featured Regional Coordinators:"

2. Great

3. I prefer the mosaic 

4. We have links to each tab and it would be great to have a one directly to the bios as well. 

5. Yes. Great idea. Please do mix it up. Order of names does not matter.

**Comment by taisdesouzalessa** [comment]
Thanks for the answers, Julia.

Here are the latest mockups:
https://redpen.io/p/nj4f7768ca3e7b79cf

@jvallera @xmatthewx @hannahkane please take a look and let me know if there is any additional feedback (add it to RedPen). 

If it is good to go please let me know here so I can move to next stage: provide assets to Gavin.

**Comment by hannahkane** [comment]
Great work, @taisdesouzalessa!

Added a couple comments to the redpen, but no blockers. +1 from me to advance to implementation.

**Comment by jvallera** [comment]
Looks great! just made a comment on Asma's bio, but all else is good to go.

**Comment by taisdesouzalessa** [comment]
Here is the latest version: https://redpen.io/p/nj4f7768ca3e7b79cf

**Comment by taisdesouzalessa** [comment]
Closing this ticket. Implementation stage is in  #2280.
